content/02.introduction.md

```diff
@@ -2,5 +2,94 @@
+This process can be broken down into three different categories: (1) the use of natural language processing techniques such as manually crafted rules and identifying key text patterns for relationship extraction, (2) the use of unsupervised methods via co-occurrence scores or clustering, and (3) supervised or semi-supervised machine learning using annotated datasets for classification of documents or sentences.
+In this section, we discuss selected efforts for each type of edge that we included in this project.
+
+#### Disease-Gene Associations 
+
+Substantial effort for Disease-associates-Gene (DaG) extraction comprised of using manually crafted rules and unsupervised methods.
+One study used hand crafted rules on generated dependency trees to extract Disease-associates-Gene relationships [@tag:pkde4j].
+These rules were based off of a sentence's grammatical structure and labeled any structure match as positive.
+We used some of these rules as inspiration for our DaG text pattern label functions.
+Another study used a co-occurrence approach to score the likelihood of association between disease and gene pairs [@tag:diseases].
+These scores were calculated using the frequency of disease and gene mentions within abstracts and within sentences.
+The results of this study was incorporated into Hetionet v1 and used as one of our distant supervision label functions.
+Another approach built off of the previously mentioned approach by using a distant supervision technique with a logistic regression model to improve the frequency calculations [@tag:cocoscore]. 
+We incorporate this work as a baseline comparison for our discriminative model (Results Section: Reconstructing Hetionet).
+Besides the mentioned two studies, other work have used pure co-occurrence for extraction [@tag:lgscore; @tag:polysearch; @tag:full_text_co_abstracts] or a combination of scores with other features to mine the DaG relationship [@tag:dg_text_pubmed ].
+Along with co-occurrence, one study used a bi-clustering approach to detect DaG relevant sentences from Pubmed abstracts [@tag:global_network].
+This clustering approach used dependency paths to group similar sentences together.
+We used the results of this work for our domain heuristic label functions.
+The benefit of using the above approaches is inessential need for annotated data.
+These methods provide great recall performance in terms of extraction, but can suffer in precision performance.
+
+Supervised machine learning for DaG has prospered with the creation of hand crafted datasets [@tag:befree; @tag:eu_adr; @tag:comagc; @tag:craft].
+This datasets served as a gold standard for tuning and testing machine learning models in DaG extraction.
+Copious amount of effort used support vector machines to detect DaG relationships [@tag:befree; @tag:dtminer; @tag:ensemble_svm].
+This model performs well in large feature spaces, but perform slowly as the number of data points increases.
```

Comment:
What do you mean perform slowly? Make predictions slowly? Train slowly? something else? Can you re-word for clarity?